apps/frontend/src/dashboard/__tests__/api.test.tsx

```diff
@@ -0,0 +1,47 @@
+import nock from 'nock';
+import { API_BASE_URL } from '../../config';
+import { getDashboard } from '../api';
+
+jest.mock('../../config');
+
+describe('getDashboard', () => {
+  afterEach(() => {
+    expect(nock.isDone()).toBe(true);
+  });
+
+  afterEach(() => {
+    nock.cleanAll();
+  });
```

Comment:
```suggestion
  afterEach(() => {
    expect(nock.isDone()).toBe(true);
    nock.cleanAll();
  });
```

I know this is counter to what you were told in the backend but this will work here and much more readable.

Even if expect(nock.isDone()).toBe(true); fails at worst it will fail the next test in addition but its probably overwritten anyway.